src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -57,11 +57,15 @@ def chunk_iter(inputs, feature_extractor, chunk_len, stride_left, stride_right):
         # add start and end paddings to the chunk
         chunk = inputs[i : i + chunk_len]
         processed = feature_extractor(chunk, sampling_rate=feature_extractor.sampling_rate, return_tensors="pt")
+
+        _main = chunk.shape[0]
         _stride_left = 0 if i == 0 else stride_left
         is_last = i + step + stride_left >= inputs_len
         _stride_right = 0 if is_last else stride_right
+        if "input_features" in processed:
+            processed["input_to_features_ratio"] = _main / processed["input_features"].shape[1]
```

Comment:
Since in that case we are already reducing the size of the sequence length, we start saving the ratio here.

It could probably be calculated from feature_extractor directly but the calculation is likely to be non trivial given all the options I have seen.